<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>1200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>1200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QTabWidget" name="tabs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1171</width>
      <height>1171</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="Sniffing">
     <attribute name="title">
      <string>Sniffing</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1141</width>
        <height>1101</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="12" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QTextBrowser" name="textBrowserDump">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>2</horstretch>
             <verstretch>2</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextBrowser" name="textBrowserRaw">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>3</horstretch>
             <verstretch>2</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>4</number>
         </property>
         <item alignment="Qt::AlignRight">
          <widget class="QCheckBox" name="Sniff_chk_monitorMode">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 14pt &quot;맑은 고딕&quot;;</string>
           </property>
           <property name="text">
            <string>Monitor Mode</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Sniff_toggleButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 14pt &quot;맑은 고딕&quot;;</string>
           </property>
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="8" column="0">
        <widget class="QTableWidget" name="tableWidget">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <column>
          <property name="text">
           <string>No</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Time</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Source</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Destination</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Protocol</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Length</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Content</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QTreeWidget" name="treeWidget">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QPushButton" name="Sniff_setButton">
         <property name="styleSheet">
          <string notr="true">font: 75 24pt &quot;맑은 고딕&quot;;</string>
         </property>
         <property name="text">
          <string>Set Target</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="Spoofing">
     <attribute name="title">
      <string>Spoofing</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1141</width>
        <height>1111</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="Spoof_toggleButton">
           <property name="styleSheet">
            <string notr="true">font: 75 24pt &quot;맑은 고딕&quot;;</string>
           </property>
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tableWidget_3">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <column>
            <property name="text">
             <string>No</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Time</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Source</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Destination</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Protocol</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Length</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Content</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBar_2">
           <property name="styleSheet">
            <string notr="true">font: 75 14pt &quot;맑은 고딕&quot;;</string>
           </property>
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Spoof_KRACKButton">
           <property name="styleSheet">
            <string notr="true">font: 75 24pt &quot;맑은 고딕&quot;;</string>
           </property>
           <property name="text">
            <string>Start KRACK</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="KRACK">
     <attribute name="title">
      <string>KRACK</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>1141</width>
        <height>1001</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTreeWidget" name="treeWidget_2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar_3">
         <property name="styleSheet">
          <string notr="true">font: 75 14pt &quot;맑은 고딕&quot;;</string>
         </property>
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="KRACK_toggleButton">
         <property name="styleSheet">
          <string notr="true">font: 75 24pt &quot;맑은 고딕&quot;;</string>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="radio_buttons">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1141</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Execution Type</string>
      </property>
      <widget class="QRadioButton" name="radio_Replay_Broadcast">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>0</y>
         <width>401</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;맑은 고딕&quot;;</string>
       </property>
       <property name="text">
        <string>Test-Replay-Broadcast</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radio_Test_GTK">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>0</y>
         <width>181</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;맑은 고딕&quot;;</string>
       </property>
       <property name="text">
        <string>Test-GTK</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radio_Test_KRACK">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>0</y>
         <width>191</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;맑은 고딕&quot;;</string>
       </property>
       <property name="text">
        <string>Test-KRACK</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radio_KRACK_Attack">
       <property name="geometry">
        <rect>
         <x>770</x>
         <y>0</y>
         <width>211</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;맑은 고딕&quot;;</string>
       </property>
       <property name="text">
        <string>KRACK-Attack</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radio_Entire">
       <property name="geometry">
        <rect>
         <x>1010</x>
         <y>0</y>
         <width>151</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;맑은 고딕&quot;;</string>
       </property>
       <property name="text">
        <string>Entire</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
